setup.py

```diff
@@ -28,12 +27,4 @@
                  runtime_library_dirs=[epics_lib,numpy_lib],
                  libraries=['Com', 'ca'])
 
-
-
-setup(name='pyca',
-      version=versioneer.get_version(),
-      cmdclass=versioneer.get_cmdclass(),
-      description='python channel access library',
-      packages=['psp'],
-      ext_modules=[pyca],
-     )
+setup(ext_modules=[pyca,])
```

Comment:
My cursory research suggests this is how to mix pyproject and C extensions, but I need to figure out a good way to test this

Reply:
Were you able to compile pyca?

Reply:
I thought I did, but I tried again on a fresh environment and installation failed looking for numpy at setup.  (as this setup.py file needs numpy).

Even after installing numpy setuptools can't find it.  Weird

Reply:
Ah ok I just need to do this properly through pyproject.toml.

Reply:
This was more confusing than I thought, but with [cef4cd](https://github.com/slaclab/pyca/pull/18/commits/cef4cd6f22676323c4c90298bfa05e72bade47ab) we're able to install pyca, and run the test suite (given pytest is installed)